### PR TITLE
HCAP-676 | HCAP-677 : Bug fixes and UI alignment fix

### PR DESCRIPTION
**Pull request by fwpushan**
------

1. Fix PSI-Cohort table view alignment
2. Add confirmation message
3. Fix bugs

Tickets:
 - [HCAP-676](https://freshworks.atlassian.net/browse/HCAP-676)  
 - [HCAP-677](https://freshworks.atlassian.net/browse/HCAP-677)  
 - [HCAP-892](https://freshworks.atlassian.net/browse/HCAP-892)
 - [HCAP-893](https://freshworks.atlassian.net/browse/HCAP-892)  
 - [HCAP-894](https://freshworks.atlassian.net/browse/HCAP-894)
 - [HCAP-897](https://freshworks.atlassian.net/browse/HCAP-897)
 - [HCAP-898](https://freshworks.atlassian.net/browse/HCAP-898)
 - [HCAP-901](https://freshworks.atlassian.net/browse/HCAP-901)
 - [HCAP-902](https://freshworks.atlassian.net/browse/HCAP-902)

### CODE REVIEW

File: client/src/components/participant-details/psi-cohort.js [hunk 1]
```diff
@@ -100,7 +100,23 @@ const PSIRouteTabs = ({
...
         ></Route>
         <Route exact path={TabDetails.assignCohort.path}>
-          <PSICohortTable disabled={disabled} rows={psiList} assignAction={assignAction} />
+          {disabled && (
+            <div>
+              <Box>
+                <Typography variant='h3'> Assigning Cohort</Typography>
+                <br />
+                <Typography variant='body1'>
+                  {' '}
```
Comment (dbayly-freshworks) [inline client/src/components/participant-details/psi-cohort.js:109]: If possible, this spacing should be done with CSS.

**Review by dbayly-freshworks — APPROVED** [review]
One nitpick, otherwise looks good.

File: client/src/components/participant-details/psi-cohort.js [hunk 2]
```diff
@@ -100,7 +100,23 @@ const PSIRouteTabs = ({
...
+                  This participant has been assigned cohort. To re-assign cohort. Please click on
+                  withdraw cohort to withdraw participant from cohort. After that, you can assign
+                  cohort in "Assigning Cohort"
```
Comment (Zenen-freshworks) [inline client/src/components/participant-details/psi-cohort.js:112]: Suggested Copy Change:
This participant has already been assigned a cohort. To reassign the cohort, please click on 'Withdraw Cohort' to withdraw them first.

File: client/src/pages/private/ParticipantDetailsView.js [hunk 3]
```diff
@@ -170,6 +180,62 @@ export default () => {
...
+                        </Grid>
+                      </Grid>
+                    </Grid>
+                  </Grid>
+                  <br />
+                  <Box>
+                    <Typography variant='body2'>
+                      Are you you going to assign this participant to{' '}
+                      <b>{cohortName(selectedCohort)}</b>. Please revise information before taking
+                      action
```
Comment (Zenen-freshworks) [inline client/src/pages/private/ParticipantDetailsView.js:218]: Suggested Copy Change:
Are you sure that you would like to assign this participant to [cohortName]? Please review the above information before proceeding.

**Review by Zenen-freshworks — COMMENTED** [review]
Looks great! Just a few small copy changes that I'd like to see before moving forward with it.

File: client/src/pages/private/ParticipantDetailsView.js (hunk 3 above)
Comment (fwpushan) [reply client/src/pages/private/ParticipantDetailsView.js:218]: OK, I copied it from design ...

File: client/src/pages/private/ParticipantDetailsView.js [hunk 4]
```diff
@@ -170,6 +180,62 @@ export default () => {
...
+                        </Grid>
+                      </Grid>
+                    </Grid>
+                  </Grid>
+                  <br />
+                  <Box>
+                    <Typography variant='body2'>
+                      Are you sure that you would like to assign this participant to{' '}
+                      <b>{cohortName(selectedCohort)}</b>.? Please review the above information
+                      before proceeding. action
```
Comment (Zenen-freshworks) [inline client/src/pages/private/ParticipantDetailsView.js:218]:
```suggestion
                      before proceeding.
```

File: client/src/components/participant-details/psi-cohort-table.js [hunk 5]
```diff
@@ -21,11 +20,29 @@ const useStyles = makeStyles({
   table: {
     minWidth: 650,
   },
+  innerRow: {
+    widows: '100%',
```
Comment (Zenen-freshworks) [inline client/src/components/participant-details/psi-cohort-table.js:24]: is this supposed to be 'width'?

Comment (fwpushan) [reply client/src/components/participant-details/psi-cohort-table.js:24]: Fixed

**Review by Zenen-freshworks — APPROVED** [review]

File: client/src/components/participant-details/psi-cohort-table.js [hunk 6]
```diff
@@ -21,11 +20,29 @@ const useStyles = makeStyles({
...
+  expand = false,
+  children,
+  assignAction,
+  rows,
+  expandComponent,
+  psi,
+  ...otherProps
+}) => {
+  const [isExpanded, setIsExpanded] = React.useState(expand);
+  console.log('cohorts =>', rows);
```
Comment (dbayly-freshworks) [inline client/src/components/participant-details/psi-cohort-table.js:45]: Stray console.log here.

**Review by dbayly-freshworks — APPROVED** [review]
One small nit, otherwise looks good.

File: client/src/components/participant-details/psi-cohort-table.js (hunk 6 above)
Comment (fwpushan) [reply client/src/components/participant-details/psi-cohort-table.js:45]: I'm creating another PR to address this

Comment (fwpushan) [reply client/src/components/participant-details/psi-cohort-table.js:45]: How eslint missed that?

**Pull request merged**